akka-distributed-data/src/test/java/akka/cluster/ddata/ORMapTest.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.cluster.ddata;
+
+import akka.cluster.Cluster;
+
+public class ORMapTest {
+
+  public void compileOnlyORMapTest() {
+    // primarily to check API accessibility with overloads/types
+    Cluster node1 = null;
+
+    ORMap<String, PNCounterMap<String>> orMap = ORMap.create();
+    // updated needs a cast
+    ORMap<String, PNCounterMap<String>> updated = orMap.update(node1, "key", PNCounterMap.create(), curr -> curr.decrement(node1, "key", 10));
+    updated.getEntries();
```

Comment:
No more methods to cover?

Reply:
I checked each of the /** Java API **/ ones. Didn't try the shared ones.